### DISCUSSION TITLE
Block Post back on bottom

### DISCUSSION THREAD

**Discussion by UltimateLuck** [AprilSylph/XKit-Rewritten#244] [Suggestion Box]
The new xKit Rewritten moves the block post button to the top meatballs as far as I can tell, but for long posts this requires seeing something I'm uncomfortable with and scrolling back through it all just to block it, where in the older version the post block was a somewhat large icon at the bottom. Is it possible to return 'postblock button at bottom' as a default, or have it be an alternative/secondary placement? Or perhaps move it where post time currently is, if time gets moved back to being next to the post date?
(ps thank you for all your hard work!)

**Comment by AprilSylph** [comment]
The button will remain exclusively in the meatballs menu. You can quickly skip back to the top of any post by using the `K` keyboard shortcut.

Also, since you mentioned it, Timestamps will likely be discontinued relatively soon, in favour of Tumblr's native timestamps, which were first seen on [this post by Tumblr Engineering](https://engineering.tumblr.com/post/654257188393697280/tumblr-hack-day-june-2021-edition) and announced as planned for sure on [this post on the Changes blog](https://changes.tumblr.com/post/661325788557246465/friday-september-3rd-2021).
